src/Init/Data/BitVec/Lemmas.lean

```diff
@@ -6182,6 +6195,65 @@ theorem toNat_lt_two_pow_sub_clz {x : BitVec w} :
         · simp [show w + 1 ≤ i by omega]
       · simp; omega
 
+/-! ### Count trailing zeros -/
+
+@[simp]
+theorem ctz_eq_reverse_clz {x : BitVec w} :
+    x.ctz = (x.reverse).clz := by
+  simp [ctz]
```

Comment:
```suggestion
theorem ctz_eq_reverse_clz {x : BitVec w} :
    x.ctz = (x.reverse).clz := by
  simp [ctz]
```

Let's un-simp this. If it is `simp`, it makes it hard to write API around `x.ctz`, as `simp` will instantly remove it into a `x.reverse.clz`, and then we lose the ability to reason about its behaviour.